tests/src/test/scala/common/WskTestHelpers.scala

```diff
@@ -280,9 +280,13 @@ trait WskTestHelpers extends Matchers {
     }
   }
 
-  def removeCLIHeader(response: String): String = response.substring(response.indexOf("\n"))
+  def removeCLIHeader(response: String): String = {
+    if (response.contains("\n")) response.substring(response.indexOf("\n")) else response
+  }
 
-  def getJSONFromCLIResponse(response: String): JsObject = removeCLIHeader(response).parseJson.asJsObject
+  def getJSONFromResponse(response: String, isCli: Boolean = false): JsObject = {
+    if (isCli) removeCLIHeader(response).parseJson.asJsObject else response.parseJson.asJsObject
```

Comment:
Seems like you can just call `removeCLIHeader()` without the need for the `isCli` flag.

Reply:
Since the Cli tests will still exist for some time, we need to make sure Cli tests also pass.

Reply:
Ah, I see. The newline from the REST call might be anywhere in the output.